### ISSUE TITLE
Flashlight doesn't work with Spotlight v972 (OSX 10.11 beta6)

### ISSUE THREAD

**Issue by NuclleaR** [nate-parrott/Flashlight#545]
It is expected to support this os and this spotlight version?

**Comment by edouardjamin** [comment]
Please see https://github.com/nate-parrott/Flashlight/issues/537

**Comment by brechtm** [comment]
I understand it should be possible to get Flashlight working in OS X 10.11 El Capitan, once SIMBL is installed as suggested in https://github.com/nate-parrott/Flashlight/issues/537#issuecomment-125895020

I installed SIMBL, but I could not get Flashlight working. On injecting SIMBL into Spotlight (see https://gist.github.com/selaromi/59666a71387d39b5bfac), I get the following error:

> Spotlight 1.0 (v972.9) has not been tested with the plugin SpotlightSIMBL 1.0 (v17). As a precaution, it has not been loaded. Please contact the plugin developer for further information.

If I change SIMBLTargetApplications/MaxBundleVersion to 973, injecting SIMBL seems to work. However, Spotlight crashes when I try to use it.

I suspect at least some changes to Flashlight are necessary in order to get it to work with Spotlight v972, but I have no idea where to start. I'm hoping someone with a clue can help.

**Comment by hgpt** [comment]
I've tried doing the

**Comment by Leomund** [comment]
@brechtm I have the same problem, did you found a way to solve it?

**Comment by brechtm** [comment]
No, haven't looked into it. As I'm not familiar with SIMBL/Flashlight code, this may take a lot of time, so I'm not really motivated.

**Comment by MrLoh** [comment]
Just accept it spotlight is dead. See #537!
